### DISCUSSION TITLE
getShipment

### DISCUSSION THREAD

**Discussion by Cohen-11** [amzn/selling-partner-api-models#3857] [General]
The getShipments, getShipmentItems ByShipmentId, getShipmentItems, and other interfaces will be deprecated in the sp API interface.
![企业微信截图_1714119232956](https://github.com/amzn/selling-partner-api-docs/assets/58458804/9a9fa09e-61fe-454d-99ec-fd8504cca39a)
I am currently using these interfaces to synchronize FBA shipments to the internal ERP system. If these interfaces are discarded, which corresponding APIs should I use to synchronize the shipment?
Currently, it is understood that there are the following interfaces:
![企业微信截图_17141193366912](https://github.com/amzn/selling-partner-api-docs/assets/58458804/864685d3-672f-4e93-8e1b-39f5faad09e0)

**Comment by oh0123** [comment]
same confusion on the interfaces update
